docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.7"
+services:
+   sync:
+     build: .
+     depends_on:
+       - "mysql"
+     ports:
+# CHANGE TO MATCH YOUR CONFIG/PROXY
+       - "8980:8080"
+       - "1337:1337"
+       - "8443:8443"
+     volumes:
+# CHANGE TO IMPORT YOUR CERTS LOCATION
+       - "./certs:/etc/certs"
+# CHANGE TO IMPORT YOUR CONFIG FILE
+       - "./config.yaml:/home/syncuser/sync/config.yaml"
+#  UNCOMMENT IF YOU PLAN ON USING A REVERSE PROXY - YOU MUST UPDATE THIS TO MATCH YOUR PROXY NETWORK       
```

Comment:
I'm not sure what "proxy network" means, if it's expected that most people using this image would want a reverse proxy you could just include it by default, otherwise it might be helpful to provide an example of how it works or just exclude it if it's specific to your setup.